core/controllers/admin.py

```diff
@@ -766,14 +754,14 @@ def get(self):
 
             user_settings = user_services.get_user_settings(user_id)
             user_roles = user_settings.roles
-            topic_ids = []
+            managed_topic_ids = []
             if feconf.ROLE_ID_TOPIC_MANAGER in user_roles:
-                topic_ids = [
-                    t_r.id for t_r in topic_fetchers.get_topic_rights_with_user(
-                        user_id)]
+                managed_topic_ids = [
+                    topic_right.id for topic_right in
```

Comment:
Use "rights" instead of "topic_right". The singular "right" doesn't really make sense here.

Reply:
Done!